mono/mini/TestDriver.cs

```diff
@@ -135,11 +165,24 @@ public class TestDriver {
 			}
 		}
 
+		if (reporter != null) {
+			reporter.ReportResults (ran, nskipped, failed);
+		}
+
 		//Console.WriteLine ("Regression tests: {0} ran, {1} failed in [{2}]{3}", ran, failed, type.Assembly.GetName().Name, type);
 		return failed;
+    }
+
+	static public int RunTests (Type type, string[] args) {
+		return internalRunTests (type, args, null);
 	}
+
+	static public int RunTests(Type type, string[] args, TestDriverReporter reporter) {
+		return internalRunTests (type, args, reporter);
+	}
```

Comment:
Left over from a bigger initial change that was later stripped down to this. I will simplify moving implementation to RunTests(Type type, string[] args, TestDriverReporter reporter) and let the others call that method instead.